Application/Mappers/PortMapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Application.Dto;
+using Domain.Entities;
+
+namespace Application.Mappers
+{
+    public interface IPortMapper
```

Comment:
Prefer having single class/interface in a file.  Extract interface into a separate file